assets/training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile

```diff
@@ -8,7 +8,6 @@ RUN pip install --upgrade pip
 
 COPY requirements.txt .
 
-RUN pip install "azureml-metrics[all]==0.0.27" && \
-    pip install "azureml-evaluate-mlflow"
+RUN pip install "azureml-evaluate-mlflow=={{latest-pypi-version}}"
```

Comment:
Did we verify that with this change we are not going back to Pytorch 1.3?

Reply:
yes verified -> https://ml.azure.com/environments/acft-hf-nlp-gpu/version/12.10.2023-src-009?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&flight=itpmerge&tid=72f988bf-86f1-41af-91ab-2d7cd011db47